### ISSUE TITLE
When I try to enter to my cloud server

### ISSUE THREAD

**Issue by cbaldini** [nextcloud/android#9411]
### Steps to reproduce
1. 
2. 
3. 

### Expected behaviour
- Enter to a nc server

### Actual behaviour
- ************ CAUSE OF ERROR ************

java.lang.IllegalArgumentException: Illegal character in scheme name at index 0: 192.168.0.103:8081
	at java.net.URI.create(URI.java:848)
	at com.owncloud.android.authentication.AuthenticatorUrlUtils.normalizeScheme(AuthenticatorUrlUtils.java:103)
	at com.owncloud.android.authentication.AuthenticatorActivity.checkOcServer(AuthenticatorActivity.java:774)
	at com.owncloud.android.authentication.AuthenticatorActivity.onEditorAction(AuthenticatorActivity.java:1382)
	at android.widget.TextView.doKeyDown(TextView.java:8348)
	at android.widget.TextView.onKeyDown(TextView.java:8217)
	at android.view.KeyEvent.dispatch(KeyEvent.java:2818)
	at android.view.View.dispatchKeyEvent(View.java:13890)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.widget.ScrollView.dispatchKeyEvent(ScrollView.java:491)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1922)
	at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:453)
	at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1841)
	at android.app.Activity.dispatchKeyEvent(Activity.java:3972)
	at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.java:122)
	at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:84)
	at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:140)
	at androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent(AppCompatActivity.java:599)
	at androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent(WindowCallbackWrapper.java:59)
	at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent(AppCompatDelegateImpl.java:3068)
	at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:367)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5581)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5449)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4956)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5009)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4975)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5115)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4983)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5172)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4956)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5009)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4975)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4983)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4956)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5009)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4975)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5148)
	at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5309)
	at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3065)
	at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2608)
	at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2599)
	at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3042)
	at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:336)
	at android.os.Looper.loop(Looper.java:174)
	at android.app.ActivityThread.main(ActivityThread.java:7559)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)
Caused by: java.net.URISyntaxException: Illegal character in scheme name at index 0: 192.168.0.103:8081
	at java.net.URI$Parser.fail(URI.java:2893)
	at java.net.URI$Parser.checkChars(URI.java:3066)
	at java.net.URI$Parser.checkChar(URI.java:3076)
	at java.net.URI$Parser.parse(URI.java:3092)
	at java.net.URI.<init>(URI.java:584)
	at java.net.URI.create(URI.java:846)
	... 58 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30180090
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: Lenovo
Device: TB-X505F
Model: Lenovo TB-X505F
Id: QKQ1.191224.003
Product: TB-X505F

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: X505F_S001142_210804_ROW

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AlvaroBrey** [comment]
Duplicate of #9338

**Comment by AlvaroBrey** [comment]
Should be fixed in upcoming 3.18.1 version
